LICENSE.md

```diff
@@ -0,0 +1 @@
+Except where otherwise noted, files in this repository are licensed under the Apache License 2.0. If you use these quickstarts in your own projects, you need not comply with any of the notice requirements of the Apache License.
```

Comment:
Where did this text come from, especially the bit about not complying with things?

Reply:
I copied the license text directly from an email from one of our license experts.